metrics/eval_audio.py

```diff
@@ -0,0 +1,121 @@
+    parser.add_argument("--ground_service", type=str, default=None, help="the url where you want to get the score of ground truth")
+    # for audio evaluation
+    parser.add_argument("--audio_eval_method", type=str, default="dnsmos", choices=["dnsmos"], help="the method to evaluate audio, like DNSMOS")
+    parser.add_argument("--dst_audio", type=str, default=None, required=True, help="the path of destination audio")
+    parser.add_argument("--audio_sample_rate", type=str, default='16000', help="the sample rate of audio")
+    parser.add_argument("--audio_channel", type=str, default='1', help="the numbers of audio channels")
+    parser.add_argument("--ground_audio", type=float, default=-1, help="the audio score of a special scenario ground truth. -1 means not use ground")
+    parser.add_argument("--binarize_bound", type=float, default=0.6, help="the bound to binarize audio score")
+    # for DNSMOS
+    parser.add_argument("--dnsmos_uri", type=str, default=None, help="the uri to evaluate audio provided by DNSMOS")
+    parser.add_argument("--dnsmos_key", type=str, default=None, help="the key to evaluate audio provided by DNSMOS")
+
+    return parser
+
+
+def get_audio_score(args):
+    eval_method = None
+
+    if args.audio_eval_method == "dnsmos":
+        eval_method = AudioEvalMethodDNSMOS(args.dnsmos_uri, args.dnsmos_key)
+    else:
+        raise ValueError("Not supoort such method to evaluate audio")
+        
+    audio_eval_tool = AudioEvaluation(eval_method, args)
+    audio_out = audio_eval_tool.eval(args.dst_audio)
+
+    return audio_out
+
+
+def get_remote_ground(args):
+    if args.ground_service[-4:] == "json":
+        with NamedTemporaryFile('w+t') as f:
+            cmd = ["wget", args.ground_service, "-O", f.name]
+            subprocess.run(cmd, check=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+            resp = f.read()
+            if len(resp) == 0:
+                raise ValueError("Error request to ground service %s" % (args.ground_service))
+            resp = json.loads(resp)
+        if args.scenario not in resp:
+            raise ValueError("Not find scenario of %s" % (args.scenario))
+        scenario_score = resp[args.scenario]
+    else:
+        resp = requests.get("http://%s/get_ground_truth/%s" % (args.ground_service, args.scenario)).text
+        if len(resp) == 0:
+            raise ValueError("Error request to ground service %s" % (args.ground_service))
+        scenario_score = json.loads(resp)    
+
+    if "ground_video" in args:
+        args.ground_video = scenario_score["video"]
+    if "ground_audio" in args:
+        args.ground_audio = scenario_score["audio"]
+    if "ground_recv_rate" in args:
+        args.ground_recv_rate = scenario_score["recv_rate"]
+
+    return args
+
+
+if __name__ == "__main__":
+    parser = init_audio_argparse()
+    args = parser.parse_args()
+    if args.scenario and args.ground_service:
```

Comment:
just want to know where the service is. Can you provide its URL and introduction in some comments?